Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3680,9 +3680,6 @@ private static void InitializeInPlayer(IInputRuntime runtime = null, InputSettin
             if (settings == null)
                 settings = Resources.FindObjectsOfTypeAll<InputSettings>().FirstOrDefault() ?? ScriptableObject.CreateInstance<InputSettings>();
 
-            if (actions == null)
```

Comment:
Removing this would completely bypass player loading of preloaded asset? Cannot see how this change is valid and would like to revert it. This area anyway need to be addressed additionally since there are existing problems with this an InputForUI integration. There are also related problems to static object stack push/pop within InputSystem/InputManager.

Reply:
Good spot.

I've reveted this and fixed 2 of the tests instead, with the knowledge that this can pickup an Asset from the test itself.

I am concerned with this way of loading the InputActionAsset though. If there are multiple in the project then it seems this could load the wrong one.

Reply:
Agree that this is too brittle, seems to be the same case for InputSwttings from previously. Likely needs an internal property serialized that allows tagging it unless there is a better way or use a dedicated path. This needs a ticket and a test case.